cmd/support-notifications/res/configuration.yaml

```diff
@@ -0,0 +1,39 @@
+Writable:
+  LogLevel: INFO
+  ResendLimit: 2
+  ResendInterval: 5s
+  InsecureSecrets:
+    SMTP:
+      SecretName: smtp
+      SecretData:
+        username: username@mail.example.com
+        password: ''
+Service:
+  Host: localhost
+  Port: 59860
+  StartupMsg: "This is the Support Notifications Microservice"
+Smtp:
+  Host: smtp.gmail.com
+  Port: 587
+  Sender: jdoe@gmail.com
```

Comment:
Can we **not** put in `jdoe@gmail.com` here?

Reply:
@bnevis-i , how's this: https://github.com/edgexfoundry/edgex-go/pull/4456/commits/fd18b926304cc8dcd1f8fd5870834444f3f0aebc

Reply:
Yes.